content/en/profiler/enabling/go.md

```diff
@@ -80,7 +86,7 @@ To begin profiling applications:
 
 6. After a minute or two, visualize your profiles in the [Datadog APM > Profiler page][10].
 
-**Note**: By default, only the CPU and Heap profiles are enabled. Use [profiler.WithProfileTypes][11] to enable additional [profile types][12].
+**Note**: By default, only the CPU and Heap profiles are enabled. Use [profiler.WithProfileTypes][11] to enable additional [profile types][12]. For relevant v2 documentation, view [profiler.WithProfileTypes][22] and [profile types][23].
```

Comment:
```suggestion
**Note**: By default, only the CPU and Heap profiles are enabled. Use [profiler.WithProfileTypes][11] to enable additional [profile types][12]. For go v2, view [profiler.WithProfileTypes][22] and [profile types][23].
```

Reply:
The suggestion I put here isn't much better and it's because it's fairly difficult to link out to external references for v1 and v2 in a concise way without making v2 seem second class

To improve this, I recommend having a page that describes dd-trace-go with a `v1` and `v2` tab that allow customers to see only what's relevant for them and have this just link there. It's an extra click to get to `profiler.WithProfileTypes` or similar, but it reduces the confusion/clutter within this doc and many others